src/components/CharactersPage/DeleteCharacterModal/DeleteCharacterModal.js

```diff
@@ -27,12 +27,15 @@ const DeleteCharacterModal = ({ isModalOpen, onClose, character }) => {
   const onDelete = () => {
     client.cache.modify({
       id: client.cache.identify({
-        __typename: typeNameEnum.Query,
+        __typename: typeNameEnum.query,
         id: "ROOT_QUERY",
       }),
       fields: {
         allPeople(existing) {
-          const characterToDelete = `${typeNameEnum.Person}:${character.id}`
+          const characterToDelete = client.cache.identify({
+            __typename: typeNameEnum.person,
+            id: character.id,
+          })
```

Comment:
Yous should actually be passing in the entire `character` object and let Apollo do it's thing instead of only providing these two properties

Reply:
Same applies for your `onSubmit` function is `CharachterForm.js`